plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/BOAndDPCompileHook.java

```diff
@@ -51,6 +45,24 @@ public void validate(Map<String, Object> toCheck) throws Exception {
             mavenConfig = getMavenConfig(config);
 
             File pluginDir = (File) moreInfo.get("pluginDir");
+            if (pluginDir.exists())
```

Comment:
I think this does not belong here, and it's causing the build to fail.

Reply:
aggggg, great catch @varyvol